project-fortis-pipeline/docs/user/admin.md

```diff
@@ -0,0 +1,88 @@
+# Admin Settings
+
+Project Fortis comes with an admin page with which you can edit site configurations and also provide details on how events should be monitored and processed. Through the admin site, you will be able to configure site settings, blacklist terms, whitelist terms (watchlist), streams (twitter, facebook, etc.), monitored places, and trusted sources.
+
+Once you deployed a site with `deploy to azure`, you may view the admin site by navigating to: http://localhost:8888/#/site/{YOUR_SITE_NAME}/admin
+
+## Site Settings
+
+Manage service tokens and set up how the site should be displayed.
+
+| Value               | Description          |
+| ------------------- | ------------- |
+| Site Name           | Used as part of the admin url.  |
+| Site Title          | Displayed on the navigation bar.|
+| Bounding Box        | By default follows `Who's on First (WOF)`. The format of the WOF bounding box is: `maxY,minX,minY,maxX`. `Divipola` is also supported.                           |
+| Header Logo Banner  | Image path of banner.  |
+| Supported Languages | A comma delimited string with languages formatted as language codes like `en` or `ar`. The admin site can translate to any of the supported languages.
```

Comment:
Should link to the full list of supported languages or to the API call that can be used to fetch the supported languages. E.g. http://docs.microsofttranslator.com/text-translate.html#!/default/get_GetLanguagesForTranslate